backend/src/main/resources/application.properties

```diff
@@ -1,7 +1,8 @@
-server.port=8081
+server.port=8082
 
 spring.jpa.database=mysql
 spring.datasource.url=${MYSQL_URL:jdbc:mysql://localhost:3306/skeleton}
+spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
```

Comment:
ahogy Roland is kiemelte, ez temp fix legyen, felveszek egy noteot ezzel kapcsolatban